Crypto/include/Poco/Crypto/EVPPKey.h

```diff
@@ -35,10 +35,8 @@
 namespace Poco {
 namespace Crypto {
 
-//@deprecated
-class ECKey;
-//@deprecated
-class RSAKey;
+class [[deprecated]] ECKey;
+class [[deprecated]] RSAKey;
```

Comment:
@obiltschnig , what shall be used instead of these?

Reply:
@matejk anything not going through EVP api is deprecated, see [this](https://github.com/pocoproject/poco/issues/3516#issuecomment-1100750944)

some serious breakage (anything non-EVP in the interfaces must go) to rectify that; I'm not sure we want to embark on that for 1.14

Reply:
Opened new ticket